packages/components/molecules/f-tabs/src/components/Tabs.vue

```diff
@@ -1,46 +1,142 @@
 <template>
     <div
-        :class="$style['c-tabs']"
-        data-test-id='tabs-component'>
-        {{ copy.text }}
+        data-test-id="tabs-component"
+        :class="$style['c-tabs']">
+        <div :class="$style['c-tabs-header']">
+            <div :class="$style['c-tabs-buttons']">
+                <button
+                    v-for="({ name, title }, i) in tabs"
+                    :key="i"
+                    :class="[{ [$style['c-tabs-button--active']]: activeTab === name }, $style['c-tabs-button']]"
```

Comment:
```suggestion
                    :class="[
                        (activeTab === name ? $style['c-tabs-button--active' : ''),
                        $style['c-tabs-button']
                    ]"
```

Reply:
Unsure as to the benefit of `(activeTab === name ? $style['c-tabs-button--active' : '')`, would you be able to explain?

Reply:
It's ultimate styliistic preference, but the reasons I prefer it are that it puts the conditional at the front (`if X then Y` rather than `Y if X`) and it is a little cleaner to me with the amount of brackets needed (you don't need to wrap the `$style` in extra `[]` using this way)

Reply:
ATM i will leave the stylistic change until people agree on a format. Will adjust the rest